src/aux-server/aux-web/shared/scene/decorators/ContextPositionDecorator.ts

```diff
@@ -206,7 +207,21 @@ export function calculateObjectPositionInGrid(
         );
     }
 
+    for (let i = objectsAtPosition.length - 1; i >= 0; i--) {
+        if (
+            !calculateBooleanTagValue(
+                context,
+                objectsAtPosition[i],
+                'aux.stackable',
+                true
+            )
+        ) {
+            totalScales = 0;
+        }
+    }
```

Comment:
@shaneYeti When `aux.stackable` is set to `false` on "this" bot we can simply skip looking for other bots. Basically `aux.stackable == false` means "ignore all other bots with respect to positioning". 

So when `aux.stackable` is false on the current bot we can skip `objectsAtContextGridPosition()` and calculating the total scales, leaving those values at their defaults of `0`. The result is that the position of a bot with `aux.stackable` set to false is equal to `localPosition + gridHeight`.

Doing this would make bots with `aux.stackable` set to false update faster than other bots since the slowest part of positioning at the moment is the `objectsAtContextGridPosition()` calculation.

Reply:
sure, I can change and test that